test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml

```diff
@@ -59,6 +59,7 @@ n_monitor_nodes: 1
 round_robin: true
 
 instance_type_db: 'i3en.3xlarge'
+gce_instance_type_db: 'n1-highmem-16'
```

Comment:
does n1-highmem-16 is equal by parameters to i3en.3xlarge?

Reply:
It is slightly bigger and faster; should be fine. There is no identical match for `i3en.3xlarge` in "GCP"